traced/traced.go

```diff
@@ -218,6 +228,47 @@ func (tc *TraceCollector) collectWorker() {
 	}
 }
 
+func (tc *TraceCollector) writeJsonTrace(trace string) {
+	// open the trace, read it and transcode to json
+	in, err := os.Open(trace)
+	if err != nil {
+		panic(err)
+	}
+	defer in.Close()
+
+	gzipR, err := gzip.NewReader(in)
+	if err != nil {
+		panic(err)
+	}
+	defer gzipR.Close()
+
+	out, err := os.OpenFile(tc.jsonTrace, os.O_WRONLY|os.O_TRUNC, 0644)
```

Comment:
Hum, we have no guarantee that filebeat (or whoever is processing the json file) will get to it before it's replaced. Can we instead make `jsonTrace` a directory, and spit out a new file with the same base name as the binary .tar.gz, but replacing its extension with .json?

Filebeat will then erase the file when it's been successfully processed.

Reply:
sure, we can do that

Reply:
done.